github/PullRequest.py

```diff
@@ -409,25 +409,67 @@ def create_comment(self, body, commit_id, path, position):
         """
         return self.create_review_comment(body, commit_id, path, position)
```

Comment:
Please rename `commit_id` in this method as well to `commit`:
```suggestion
        return self.create_review_comment(body, commit, path, position)
```